proto/mixer.proto

```diff
@@ -450,6 +450,14 @@ service Mixer {
     };
   }
 
+  rpc BulkObservationsSeriesLinked(datacommons.v1.BulkObservationsSeriesLinkedRequest)
+      returns (datacommons.v1.BulkObservationsSeriesResponse) {
```

Comment:
If this is an API endpoing visible to external users, I slightly feel it's better to create its own response proto message instead of reusing one, even though their contents might be the same:

BulkObservationsSeriesLinkedResponse

Reply:
1. The user visible JSON structure won't be different with current or a newly created proto if they are the same.
2. If we use a new proto for naming clearness, there would need a dummy transformation from old proto to the new proto since the "linked" API calls the bulk API. In that sense, it's easier for those APIs share the same response proto.